andela/hc-wits-kla
# #158175308 Fix frontend test

#### What does this PR do?
* Have front tests reviewed.
#### Description of Task to be completed?
Have the following tests working in `hc/front/test/` folder
* `test_invalid_form_entry` in `test_add_channel.py` file.
* `test_pushover_validates_priority` in `test_add_pushover.py` file.
* `test_team_access_works` in `test_add_channel.py` file.
#### How should this be manually tested?
* Clone this application's git branch and set up the application as instructed in the `README.md` file
* Open terminal and ensure you're in the application folder
* Run `./manage.py test`
#### What are the relevant pivotal tracker stories?
* [#158175308](https://www.pivotaltracker.com/story/show/158175308)
